packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1128,6 +1130,11 @@ class DropdownButton<T> extends StatefulWidget {
   /// instead.
   final Color? dropdownColor;
 
+  /// Padding of this widget.
```

Comment:
What part of the widget?

How is this different from the user just wrapping the `items` in padding?

Reply:
> How is this different from the user just wrapping the items in padding?

The popup menu already adds a padding to the box. If `items` are wrapped in a `Padding`, it produces a weird effect in the popup (yet, looking good on the button). On the other hand, if no `Padding` is added, the popup looks good, but the button does not.

I believe that adding the same padding added to the popup menu to the button is a good fix for the issue, which is what this PR does

Reply:
This documentation is still lacking some more context from my first comment here. :)

Reply:
Pushed a change to update the documentation.

Reply:
Thank you! It isn't actually the padding around the button though, is it? It is inside of the button.
The docs should still also include an explanation of how this is different from just wrapping the button, or the content of the button, in a Padding widget. As well as, how does the padding affect the overall size of the button? 

All of the things we have been discussing in this PR make for helpful documentation for developers. :)